_errors.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+/* eslint-disable */
```

Comment:
Unless you _really_ want to completely disable eslint you can target the offending rules at the top of the file, e.g.

```javascript
// @flow
/* eslint-disable max-len */
// Molotov Errors
module.exports = {
  // ...
};
```

Reply:
👍